src/NServiceBus.AcceptanceTesting/Support/ScenarioRunner.cs

```diff
@@ -13,7 +13,13 @@
 
     public class ScenarioRunner
     {
-        public static async Task<RunSummary> Run(RunDescriptor runDescriptor, List<IComponentBehavior> behaviorDescriptors, Func<ScenarioContext, bool> done)
+        // remove in v8
```

Comment:
I did not want to take a dependency on obsoleteex nor did I want to break existing usages in this minor release. So I figured I add now a comment.

Reply:
Perhaps also raise an issue for the removal and assign it to https://github.com/Particular/NServiceBus/milestone/193?

That way we won't forget

Reply:
Raised https://github.com/Particular/NServiceBus/issues/5214